### PR TITLE
Bence stinks

### PR DESCRIPTION
**Pull request by eugeniocataldo**
Daaaamn, he really does

### CODE REVIEW

File: sequioa/trunk.md [hunk 1]
```diff
@@ -14,3 +14,7 @@ Greatly improve resistance to overheating in fire situation.
 ### Redcuded sap content
 
 Keeping the `bark` dry means we leave less opporunities for things to catch on fire.
+
+### More permeability
+
+The trunk becomes way more permeable so it can retain better the water that falls from rain
```
Comment (Jeroen-Doornbos) [inline sequioa/trunk.md:20]: What about other water sources?

File: sequioa/crown.md [hunk 2]
```diff
@@ -10,3 +10,4 @@ Provide the support and connection between the higher level sub-modules and the
 
 Display green colour.
 Handle chemical data conversion and output.
+Draw leaves in the cutest and hottest curly shape that would make us sell many ~~C-Traps~~ trees
```
Comment (Jeroen-Doornbos) [inline sequioa/crown.md:13]: Wouldn't the C-trap than be the Tree-sap?

File: sequioa/trunk.md [hunk 3]
```diff
@@ -14,3 +14,7 @@ Greatly improve resistance to overheating in fire situation.
 ### Redcuded sap content
```
Comment (Jeroen-Doornbos) [inline sequioa/trunk.md:14]: there is a spelling error here

File: sequioa/trunk.md (hunk 1 above)
Comment (Jeroen-Doornbos) [inline sequioa/trunk.md:20]: what about other water sources

File: sequioa/trunk.md [hunk 4]
```diff
@@ -14,3 +14,7 @@ Greatly improve resistance to overheating in fire situation.
...
 
 Keeping the `bark` dry means we leave less opporunities for things to catch on fire.
+
+### More permeability
```
Comment (Jeroen-Doornbos) [inline sequioa/trunk.md:18]: this name needs to be more permeable

File: sequioa/crown.md (hunk 2 above)
Comment (Jeroen-Doornbos) [inline sequioa/crown.md:13]: suggestion: 
Use _wavy_ instead of _curly_ for more hotness

**Comment by bkarpati** [comment]
Thanks for participating in the Git workshop. I'm going to close this PR for now and also delete the remote branch.
Please stay tuned for the follow-up.

**Pull request closed without merging**